Database/DatabaseSender.py

```diff
@@ -48,7 +49,7 @@ def create_courses(self):
             credit VARCHAR(255),
             info VARCHAR(2000),
             offering VARCHAR(255),
-            online BOOLEAN,
+            is_online INT,
```

Comment:
Why int?

Reply:
Oracle does not support BOOLEAN so it needs to be an int

Reply:
Also 'online' is a reserved keyword so I changed it to 'is_online'